cf/read_write/um/umread.py

```diff
@@ -2000,7 +2000,7 @@ def create_data(self):
 
                 indices = [(i, rec) for i, rec in enumerate(recs)]
 
-                if z_axis in self.down_axes:
+                if nz > 1 and z_axis in self.down_axes:
```

Comment:
Oops, we should have made this edit as part of the fix for https://github.com/NCAS-CMS/cf-python/issues/641 (I explicitly suggested it on the Issue :woman_facepalming: ). Sorry I didn't spot that in review for that PR! Either way, getting it added now is better than never :slightly_smiling_face: